retention bytes command -1 flag updated desc. (#1456)
redhat-developer/app-services-cli: docs/commands/rhoas_kafka_topic_create.md

      --name string             Topic name
  -o, --output string           Specify the output format. Choose from: "json", "yaml", "yml"
      --partitions int32        The number of partitions in the topic (default 1)
      --retention-bytes int     The maximum total size of a partition log segments before old log segments are deleted to free up space

>                                 (default -1 as no retention size limit is assigned by default)

Reviewer: ```suggestion
                                (value of "-1" is set by default indicating no retention size limits.)
```

Reviewer 2: Suggestion looks good.
